Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.4.4-slim
+
+WORKDIR /protein
+
+RUN apt-get update -qq && apt-get install -y curl netcat
+
+RUN curl -qO https://raw.githubusercontent.com/eficode/wait-for/f71f8199a0dd95953752fb5d3f76f79ced16d47d/wait-for && chmod a+x ./wait-for
+
+COPY . ./
+
+RUN bundle install
+
+CMD ["/bin/bash"]
```

Comment:
Is this really needed for anything?

If so then fine, just wondering. Looks like `docker(-compose) run` would replace this easily for debug sessions.

Reply:
I have removed it. It simply didn't matter. `docker-compose.yml` overrides this command anyway.